app/models/customer.rb

```diff
@@ -75,6 +75,36 @@ def require_driver_form?
     !driver_form_content.blank?
   end
 
+  # returns Human readable string of the validity length of driver training
+  def helper_driver_training_validity_length?
```

Comment:
This is a good start, but there are some complexities one needs to consider when working with production data.

For example, many customers will never set a validity length, so if `driver_training_validity_length` is `nil`, this method will fail as you can't compare 90 with `nil`.

Also, we might change the dropdown options on the edit page at some point, so ideally this method shouldn't depend on the options in that menu and should accept any positive or negative integer.